gdal/doc/source/index.rst

```diff
@@ -11,6 +11,7 @@ This documentation is also available as a `PDF file <gdal.pdf>`_.
 
     download
     programs/index
+    samples
```

Comment:
Given they are "random quality/unsupported code", I would not put that at top level of the documentation. More as a paragraph of the python page

Reply:
> Given they are "random quality/unsupported code", I would not put that at top level of the documentation. More as a paragraph of the python page

Not sure what you mean by "as a paragraph of the python page".
Maybe as a link from the programs level ?

Reply:
I thought in https://gdal.org/api/python.html. Actually there's some mention of them already

Reply:
I was thinking that the samples page could link to sample scripts in all the supporting languages.

Reply:
I'd probably keep the samples per language. Pythoners are probably not keen at looking at C# examples and vice-versa.

Reply:
> I'd probably keep the samples per language. Pythoners are probably not keen at looking at C# examples and vice-versa.

I see your point, but is there a page that points out where are the sample scripts in each language?

Reply:
I put python_samples just under python_gotchas. It seems like the right place :)